src/style.css

```
@@ -1 +1,85 @@
 /* Styles go here */
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  margin: 0;
+
+  --blue-color: #00acdc;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  max-width: 1200px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo {
+  display: inline-flex;
+
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  /* gap: 20px; */
+
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: 60px;
+  white-space: nowrap;
+}
+
+.nav__link:not(.is-active) {
+  color: #000;
+}
+
+.is-active {
+  color: var(--blue-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  background-color: var(--blue-color);
+  border-radius: 8px;
+
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

width: 100% instead

---

```suggestion
  position: absolute;
  bottom: 0;
  width: 100%;
```
this one doesn't work as expected

```suggestion
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%
```
in this occasion it's ok

May I know why using left: 0 + width: 100% is more acceptable than using left and right: 0 in order to cover the entire available space?

https://ibb.co/Nrng0Pw

---

both options are ok, Bohdan's option probably more obvious